### DISCUSSION TITLE
How to run Stalwart without the Caddy community container ?

### DISCUSSION THREAD

**Discussion by AiroPi** [nextcloud/all-in-one#4615] [Questions] [answered]
I have already have a reverse proxy for my server and can't setup the caddy community container, but this one seems *required* by Stalwart which doesn't run at all without these containers. Also, why does Stalwart need to access to the certificates ? I don't know if I will be able to make it access the certificates from my own reverse proxy :l

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/pull/4556
